index.html

```diff
@@ -359,41 +358,28 @@ <h1 class="greetings2">About Me</h1>
       </div>
       <div class="from-details">
         <!-- Get In Touch Form STARTs here-->
-        <form  id="get-in-touch" action="https://formspree.io/f/mnqydzlw" method="post">
+        <form id="get-in-touch" action="https://formspree.io/f/mnqydzlw" method="post">
```

Comment:
[OPTIONAL] Sticking to the design is a requirement in this milestone, kindly see that your form section looks a bit different from the design😉😉 I suggest you kindly fix it as sticking to the design in a project cycle shows a high level of professionalism😉

yours
<img width="931" alt="yours" src="https://user-images.githubusercontent.com/64643995/221373336-128abfa5-24c0-498d-852b-c03ba05858e9.png">

design
<img width="508" alt="design" src="https://user-images.githubusercontent.com/64643995/221373356-2e510bd5-215e-4fb0-a353-589b023bb164.png">